core/src/chiplets/hasher.rs

```diff
@@ -34,6 +34,9 @@ pub const STATE_COL_RANGE: Range<usize> = create_range(ROW_COL_IDX + 1, STATE_WI
 /// Number of field elements in the capacity portion of the hasher's state.
 pub const CAPACITY_LEN: usize = STATE_WIDTH - RATE_LEN;
 
+/// The element of the capacity register where the domain is set when initializing the hasher.
+pub const CAPACITY_DOMAIN_ELEMENT: usize = 1;
```

Comment:
nit: I would name and document this as `index` rather than `element` (and in that case we usually use `idx`)